contrib/runners/mistral_v2/query/mistral_v2.py

```diff
@@ -101,11 +101,13 @@ def _get_workflow_result(self, exec_id):
                 raise exceptions.ReferenceNotFoundError(mistral_exc.message)
             raise mistral_exc
 
-        result = jsonify.try_loads(execution.output) if execution.state in DONE_STATES else {}
+        result = jsonify.try_loads(
```

Comment:
Now back to the disable no-member pragma - just curious why this is needed now, but it wasn't needed in the past.